.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: node_js
+
 node_js:
   - "14"
```

Comment:
we should consider adding multiple node versions here. Travis will do a build for each version listed, and since we are running some of the packages on Portal which is still using node 10, I think we should keep that version at least. Furthermore, I would like to add all LTS versions of node here. This way we can say we support `LTS`.

Reply:
agreed, let's go for `12.x` and `14.x`